db/migrations/04_add_user_id_column.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE peeps ADD COLUMN user_id INTEGER REFERENCES users;
```

Comment:
Stepwise table editing. I like this, it shows how you worked through the process rather than just going back and editing the create table file.